test/test_collection.py

```diff
@@ -217,6 +218,27 @@ def test_imports(self):
         from pymongoexplain import ExplainableCollection
         self.assertEqual(ExplainableCollection, ExplainCollection)
 
+    def test_verbosity(self):
+        res = self.explain.find({})
+        self.assertNotIn("executionStats", res)
+        self.assertNotIn("allPlansExecution", res.get("executionStats", []))
+        self.explain = ExplainCollection(self.collection, verbosity="executionStats")
+        res = self.explain.find({})
+        self.assertIn("executionStats", res)
+        self.assertNotIn("allPlansExecution", res["executionStats"])
+        self.explain = ExplainCollection(self.collection, verbosity="allPlansExecution")
+        res = self.explain.find({})
+        self.assertIn("executionStats", res)
+        self.assertIn("allPlansExecution", res["executionStats"])
+
+    def test_comment(self):
+        self.explain.find({})
+        self.assertNotIn("comment", self.logger.cmd_payload)
+        self.explain = ExplainCollection(self.collection, comment="hey, "
+                                                                  "I'm a "
+                                                                  "comment")
```

Comment:
Please fix the string format here. 3 short strings on new lines makes this harder to read. How about:
```
self.explain = ExplainCollection(self.collection, comment="comment")
```

Reply:
Done.